app/views/sessions/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<% provide :title, t(".sign-in") %>
+<h1><%= t ".text1" %></h1>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for(:session, url: login_path) do |f| %>
+
+      <%= f.label :email, t(".email") %>
+      <%= f.email_field :email, class: 'form-control' %>
+
+      <%= f.label :password, t(".pass") %>
```

Comment:
e tìm hiểu i18n for label rồi sửa lại nha

Reply:
ok a